src/modules/get_professors_by_class/app/get_professors_by_class_viewmodel.ts

```diff
@@ -0,0 +1,21 @@
+
+import { User } from '../../../shared/domain/entities/user';
+
+
+export class GetProfessorsByClassViewModel {
+  private professors: { [key: string]: { name: string, email: string, RA: string } } = {};
```

Comment:
add message "professors by class returned" (it was not on the docs but I added it now)